docs/source/manifest_generation.rst

```diff
@@ -0,0 +1,177 @@
+     - Options include `"excel"` or `"google_sheet"`.
+
+This will generate a manifest directly from the API.
+
+
+Generate a manifest using a dataset on synapse
+----------------------------------------------
+
+Option 1: Use the CLI
+~~~~~~~~~~~~~~~~~~~~~~
+
+.. note::
+
+    Ensure your **Synapse credentials** are configured before running the command.
+    You can obtain a **personal access token** from Synapse by following the instructions here:
+    `<https://python-docs.synapse.org/tutorials/authentication/#prerequisites>`_
+
+
+The **top-level dataset** can be either an empty folder or a folder containing files.
+
+   .. code-block:: bash
+
+       schematic manifest -c /path/to/config.yml get -dt <your_data_type> -s -d <synapse_dataset_id>
+
+   - **-c /path/to/config.yml**: Specifies the configuration file containing the data model location and asset view (`master_fileview_id`).
+   - **-dt <your_data_type>**: Defines the data type/schema model for the manifest (e.g., `"Patient"`, `"Biospecimen"`).
+   - **-d <your_dataset_id>**: Retrieves the existing manifest associated with a specific dataset on Synpase.
+
+Option 2: Use the API
+~~~~~~~~~~~~~~~~~~~~~~
+
+To generate a manifest using the **Schematic API**, follow these steps:
+
+1. Locate the **`manifest/generate`** endpoint in the **Swagger UI**.
+2. Click **"Try it out"** to enable input fields.
+3. Enter the required parameters and execute the request:
+
+   - **schema_url**: The URL of your data model.
+     - If your data model is hosted on **GitHub**, the URL should follow this format:
+       - JSON-LD: `https://raw.githubusercontent.com/<your-repo-path>/data-model.jsonld`
+       - CSV: `https://raw.githubusercontent.com/<your-repo-path>/data-model.csv`
+
+   - **output_format**: The desired format for the generated manifest.
+     - Options include `"excel"` or `"google_sheet"`.
+
+   - **data_type**: The data type or schema model for your manifest (e.g., `"Patient"`, `"Biospecimen"`).
+     - You can specify multiple data types or enter `"all manifests"` to generate manifests for all available data types.
+
+   - **dataset_id**: The **top-level Synapse dataset ID**.
+     - This can be a **Synapse Project ID** or a **Folder ID**.
+
+   - **asset_view**: The **Synapse ID of the fileview** containing the top-level dataset for which you want to generate a manifest.
+
+Generate a manifest using a dataset on synapse and pull annotations
+--------------------------------------------------------------------
+
+Option 1: Use the CLI
+~~~~~~~~~~~~~~~~~~~~~~
+
+.. note::
+
+    Ensure your **Synapse credentials** are configured before running the command.
+    You can obtain a **personal access token** from Synapse by following the instructions here:
+    `<https://python-docs.synapse.org/tutorials/authentication/#prerequisites>`_
+
+
+The **top-level dataset** can be either an empty folder or a folder containing files.
+
```

Comment:
This seems a little vague. Should we expand on this?

Reply:
See https://github.com/Sage-Bionetworks/schematic/pull/1583/files#r2021885467 about linking off to that documentation.

What would also be useful there is what it means to be a "top-level" dataset vs ...? A non-top level dataset? This is implying the hierarchical nature of your project data (https://schematicpy.readthedocs.io/en/schematic-252_a/index.html#project-data-layout), maybe there is something that could be done using ascii art to show the project layout, and hierarchical vs flat layouts.

Reply:
I added some examples.